documentation/assemblies/assembly-kafka-broker-configuration.adoc

```diff
@@ -14,7 +14,7 @@
 You can specify and configure most of the options listed in the "Broker Configs" section of the {ApacheKafkaBrokerConfig}. You cannot configure options that are related to the following areas:
 
 * Security (Encryption, Authentication, and Authorization)
-* Listener configuration
+* Listener configuration (apart from installing per-broker certificates)
```

Comment:
I think this part is misleading. This section talks about broker specific config properties that you find in the official Apache Kafka documentation. There is no parameter for specifying "per-broker certificates" which is something that Strimzi adds. The sentece in parenthesis let me think that I can, in this `config` section, set "per-broker certificates" as the only listener configuration part. It's not clear that this happens in a different way and not here.

Reply:
Agree with Paolo, his think this should remain unchanged - ntohing really changed to this and you cannot configure the custom certificates in the `config` section anyway.

Reply:
Thanks for the explanation. I've removed the parentheses.